plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py

```diff
@@ -8,14 +8,12 @@
 from omegaconf import MISSING
 
 
-def default_field(obj):
-    return field(default_factory=lambda: obj)
-
-
 @dataclass
 class RayConf:
     ray_init_cfg: Dict[str, str] = field(default_factory=dict)
-    ray_remote_cfg: Dict[str, Any] = default_field({"num_cpus": 1, "num_gpus": 0})
+    ray_remote_cfg: Dict[str, Any] = field(
+        default_factory=lambda: {"num_cpus": 1, "num_gpus": 0}
+    )
```

Comment:
why are those not type in a Structured Config?

Reply:
There's many params can be passed in to [`ray.init()`](https://github.com/ray-project/ray/blob/master/python/ray/worker.py#L470-L501)

Before I was making RayInitConf extending `Dict[str, str]`, last time we met I think we agreed that it probably makes more sense to just do `Dict[str, str]` directly.

Reply:
You resolved it without responding.
1. Is this something you considered and rejected?
2. Is it something you considered and will look at later?